src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -175,12 +176,6 @@ public static void importProject(RemoteRobot remoteRobot, String projectsPath, S
         // Note that at specific points in time, the window pane items will re-arrange themselves
         // as content is displayed. This, has an effect on the location of the items on the frame.
         ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofMinutes(2));
-        ComponentFixture fileMenuEntry = projectFrame.getActionMenu("File", "60");
-        RepeatUtilsKt.waitFor(Duration.ofSeconds(30),
-                Duration.ofSeconds(1),
-                "Waiting for the File action menu on the main window pane to be enabled",
-                "The file action menu on then main window pane is not enabled",
-                () -> projectFrame.isComponentEnabled(fileMenuEntry));
```

Comment:
I removed this because the `File` menu is only active when we click on the `Main Menu` (the four lines in the top bar).